website/docs/releases.mdx

```diff
@@ -37,21 +61,18 @@ config
 - Weave GitOps default admin user can now “read” all objects. Why is this important? As users are trying out Weave GitOps they will most likely try it out with some of their favorite Cloud Native tools such as Crossplane, Tekton, Istio, etc. This enables them to see all of those resources and explore the full power of Weave GitOps. We still do not recommend this user for “production-use” cases, and customers should always be pushed towards implementing OIDC with scoped roles.
 
 #### Pipelines - adding Pipelines through Templates
-- From the Pipelines view you can add new Pipelines in a way which leverages GitOpsTemplates, additionally - to help users configure these, we’ve provided documentation which has some samples.
+- From the Pipelines view you can add new Pipelines in a way which leverages GitOpsTemplates, additionally - to help users configure these, we’ve provided [documentation](.gitops.weave.works/docs/pipelines/pipeline-templates/) which has some samples.
 
-#### Multi-Flux
-- We can now support users running multiple instances of Flux on a single cluster for resource isolation, note the two versions within different namespaces on the cluster flux-system/dev
+#### Support for multiple Flux instances on a single cluster
+- Support for running multiple flux instances in different namespaces on a single cluster for better resource isolation.
```

Comment:
nit: remove the word "better", it's not a clear recommendation to do this, its just an option.

Reply:
done.